konf/site.yaml

```diff
@@ -104,6 +105,9 @@ production:
     if ($host = 'apps.ubuntu.com' ) {
       rewrite ^ https://snapcraft.io/ permanent;
     }
+    if ($host = 'buy.ubuntu.com' ) {
+      rewrite ^ https://ubuntu.com/advantage$request_uri? permanent;
```

Comment:
@nottrobin should it be without the `$request_uri?` in this case?

eg: https://buy.ubuntu.com/collections/ubuntu-advantage-virtual will 404 in https://ubuntu.com/advantage/collections/ubuntu-advantage-virtual